packages/blocks/src/api/parser/apply-built-in-validation-fixes.js

```diff
@@ -15,11 +16,22 @@ import { fixCustomClassname } from './fix-custom-classname';
  * @return {WPBlock} Fixed block object
  */
 export function applyBuiltInValidationFixes( block, blockType ) {
```

Comment:
If this approach works, we may be able to apply a similar approach to the anchor support as well (See #51402).

This means adding a new function like `fixAnchor()` here in the future.

Reply:
I wonder if it would be more performant to have a single `fixGlobalAttributes` method that fixes all these cases instead of multiple parsers.

This isn't a blocker; it's just something to keep in mind after everything is merged, including the `fixAnchor` solution.